cypress/integration/Buy.spec.js

```diff
@@ -292,4 +295,30 @@ describe('Buying', () => {
       summary.expectPriceOf(price);
     });
   })
+
+  context('ANY for ANY', () => {
+    it('without reaching the receive token min value', () => {
+      const from = "ETH";
+      const to = "DAI";
+
+      const minValue = settings.chain.private.tokens[to.toLowerCase()].minValue;
+
+      const trade = new Trade()
+        .sell(from)()
+        .buy(to)(0.00014);
+
+      trade.containsError(ERRORS.MINIMAL_VALUE(minValue, to));
+    });
+
+    it('without having enough trade volume for receive token', () => {
+      const from = "ETH";
+      const to = "DAI";
+
+      const trade = new Trade()
+        .sell(from)()
+        .buy(to)(10000);
+
+      trade.containsError(`No orders available to buy ${10000} ${to}`);
```

Comment:
wanna follow same pattern with `ERRORS`?

Reply:
It's not possible. IF you check the ERRORS object it returns REACT elements. The thing is that Thomas like to complicate stuff so part of the message must be bolded. I had no other idea how to do it except wrapping it into tags.